persistence/src/main/java/com/epam/esm/mapper/CertificateMapper.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.esm.mapper;
+
+import com.epam.esm.entity.Certificate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+/**
+ * The type Certificate mapper.
+ */
+@Component
+public class CertificateMapper implements RowMapper<Certificate> {
+
+    @Override
+    public Certificate mapRow(ResultSet rs, int rowNum) throws SQLException {
+        return Certificate.builder().id(rs.getLong("id")).name(rs.getString("name"))
```

Comment:
I suppose a new line for an each field will be looked better

Reply:
done!